web/blob/static/js/status.js

```diff
@@ -0,0 +1,26 @@
+/* globals PATH_PREFIX, jQuery */
+(function ($, undefined) {
```

Comment:
Note: I'm not a frontender, so expect some naive questions or comments :)

What's the second argument `undefined` here about? Given that `$` is already globally defined by jQuery, I'm more used to seeing a post-load function call just written out simply like this:

```javascript
$(function() {
  ...
});
```

Similar to https://github.com/prometheus/prometheus/blob/master/web/blob/static/js/graph.js#L679

Unless there's a specific reason to do otherwise, maybe keep it to that style for consistency instead of passing in the long-form `jQuery`?